CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Fixes
 
-- Fixed ArgumentNullException in FormRequestPayloadExtractor when handling invalid form data on ASP.NET ([#3734](https://github.com/getsentry/sentry-dotnet/pull/3734))
+- ArgumentNullException in FormRequestPayloadExtractor when handling invalid form data on ASP.NET ([#3734](https://github.com/getsentry/sentry-dotnet/pull/3734))
+- Crash when using NLog with FailedRequestStatusCodes options in a Maui app with Trimming enabled ([#3743](https://github.com/getsentry/sentry-dotnet/pull/3743))
```

Comment:
Can you add a test that repros this crash, so we make sure this PR fixes it?

Reply:
> Can you add a test that repros this crash, so we make sure this PR fixes it?

@bruno-garcia there's no easy way to test this. Enabling trimming breaks NSubstitute on the mobile targets... so the only way to test this would be an integration test (e.g. write a powershell script to create and compile an app that demonstrates this error). It seemed like more effort than it was worth so I told @bricefriha not to bother.